ee/tabby-webserver/src/schema/github_repository_provider.rs

```diff
@@ -1,9 +1,30 @@
 use async_trait::async_trait;
-use juniper::{GraphQLObject, ID};
+use juniper::{GraphQLInputObject, GraphQLObject, ID};
 
 use super::Context;
 use crate::{juniper::relay::NodeType, schema::Result};
 
+#[derive(GraphQLInputObject)]
```

Comment:
add validation logic?

Reply:
What are we validating?

Reply:
e.g:
1. display name should be a relatively display friendly name (think `name` field for git repository).
2. application_id should have minimum length requirement.
3. similar to secret.